app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -44,6 +45,13 @@ def index
 
     private
 
+    def redirect_to_not_found_if_selfie_requested_in_production
+      if Rails.env.production? &&
+         params['biometric_comparison_required'] == 'true'
+        redirect_to '/page_not_found'
```

Comment:
Looks like we have `render_not_found` in ApplicationController, I would consider using that.

Actually, I would consider using `render_not_acceptable` since I think that better matches the spirit of what we're doing here (even though the resulting error displayed to the user is the same).

Reply:
Thank you. Changed.